build.gradle

```diff
@@ -105,6 +109,12 @@ subprojects {
     artifacts {
         archives testJar
     }
+
+    javadoc {
+        // TODO audit and fix our javadocs so that we don't need this setting
```

Comment:
Can add comment that this was added because shipkit auto-builds javadoc which we were skipping before.